examples/mnist/conv_mnist.py

```diff
@@ -23,15 +26,15 @@
 
 parser = argparse.ArgumentParser()
 parser.add_argument("--seed", type=int, default=0)
-parser.add_argument("--n_train", type=int, default=60000)
+parser.add_argument("--n_epochs", type=int, default=1)
 parser.add_argument("--n_test", type=int, default=10000)
 parser.add_argument("--kernel_size", type=int, default=16)
 parser.add_argument("--stride", type=int, default=4)
 parser.add_argument("--n_filters", type=int, default=25)
 parser.add_argument("--padding", type=int, default=0)
 parser.add_argument("--time", type=int, default=50)
 parser.add_argument("--dt", type=int, default=1.0)
-parser.add_argument("--intensity", type=float, default=0.5)
```

Comment:
Is there is a reason why to change the default values?

Reply:
Bindsnet's previous implementation of MNIST provided values 0-255 while Pytorch's provides values 0-1. Hence the change from intensity 0.5 to 128, so that the values fed into the PoissonEncoder is the same.

Reply:
Thanks